src/qt/transactiontablemodel.cpp

```diff
@@ -630,7 +630,7 @@ QVariant TransactionTableModel::data(const QModelIndex &index, int role) const
     case TxPlainTextRole:
         {
             QString details;
-            QDateTime date = QDateTime::fromTime_t(static_cast<uint>(rec->time));
+            QDateTime date = QDateTime::fromSecsSinceEpoch(qint64{rec->time});
```

Comment:
`rec->time` is of type `qint64`:
https://github.com/bitcoin-core/gui/blob/e033ca13794699cf4744e71647db75c583a9a600/src/qt/transactionrecord.h#L116